god-mode.el

```diff
@@ -338,17 +338,46 @@ Appends to key sequence KEY-STRING-SO-FAR."
         (concat modifier key)))))
 
 (defun god-mode-lookup-command (key-string)
-  "Execute extended keymaps in KEY-STRING, or call it if it is a command."
-  (when key-string
-    (let* ((key-vector (read-kbd-macro key-string t))
-           (binding (key-binding key-vector)))
-      (cond ((commandp binding)
-             (setq last-command-event (aref key-vector (- (length key-vector) 1)))
-             binding)
-            ((keymapp binding)
-             (god-mode-lookup-key-sequence nil key-string))
-            (:else
-             (error "God: Unknown key binding for `%s`" key-string))))))
+		"Look up and return the command for KEY-STRING.
+Tries fallback for the last key by removing Control modifier. If no command is found,
+returns a keymap to allow further key input, or nil if completely unbound."
+		(when key-string
+			(let* ((key-vector (read-kbd-macro key-string t))
+						 (binding (key-binding key-vector)))
+				(cond
+				 ;; Case 1: found an actual command
+				 ((commandp binding)
+					(setq last-command-event (aref key-vector (1- (length key-vector))))
+					binding)
+
+				 ;; Case 2: it's a keymap (partial key sequence)
+				 ((keymapp binding)
+					(god-mode-lookup-key-sequence nil key-string))
+
+				 ;; Case 3: try fallback by removing C- from the last key
+				 ((and (string-match "\\(.*\\) C-\\(.\\)$" key-string))
+					(let* ((prefix (match-string 1 key-string))
+								 (last (match-string 2 key-string))
+								 (fallback-key (string-trim (format "%s %s" prefix last)))
+								 (fallback-vector (read-kbd-macro fallback-key t))
+								 (fallback-binding (key-binding fallback-vector)))
+						(cond
+						 ((commandp fallback-binding)
+							(message "Using fallback command: %s" fallback-key)
+							(setq last-command-event (aref fallback-vector (1- (length fallback-vector))))
+							fallback-binding)
+
+						 ((keymapp fallback-binding)
+							(message "Using fallback keymap: %s" fallback-key)
+							(god-mode-lookup-key-sequence nil fallback-key))
+
+						 (t
+							;; Not a command nor a map — wait for more input, don't fail
+							nil))))
+
+				 ;; Case 4: totally unbound
+				 (t
+					nil)))))
```

Comment:
`(t nil)` in these `cond` forms is redundant. 

You can remove them, while keeping the comments.